GVFS/GVFS.Service/GvfsService.cs

```diff
@@ -287,28 +287,37 @@ private void CheckEnableGitStatusCacheTokenFile()
             try
             {
                 string statusCacheVersionTokenPath = Path.Combine(Paths.GetServiceDataRoot(GVFSConstants.Service.ServiceName), GVFSConstants.GitStatusCache.EnableGitStatusCacheTokenFile);

```

Comment:
I think we should get rid of this file. Comparing the capability file and reboot time should be enough, shouldn't it?

Reply:
I see that you discussed this in the PR comments and left it as a TODO. I do think we should drop it, since it's another point of failure and not a necessary one, it would seem.

Reply:
@sanoursa I think we should take this PR in isolation for M139 so we can get it out the door.

That would leave us in a state where we could take the second point up in master moving forward, if we felt we needed to revisit it.

Reply:
That works for me. The file that is dropped by the installer is one that we will have to maintain forever. The status cache token file is not, so I agree we could clean it up later.

Reply:
Thanks! This approach seems reasonable.